momentum/3/3-reference/options-summary.md

```diff
@@ -6,11 +6,12 @@ description: "This section displays all configuration file options visible in th
 
 This section displays all configuration file options visible in the following scopes; global, domain, host, binding, binding_group, security, pathway, pathway_group, listener, listen, peer, threadpool, debug_flags and cluster. Module-specific options are documented in the module documentation and options specific to Mobile Momentum are documented in the Mobile Momentum documents. Listener options are listed here but for a separate table of listener-specific options see [Table 9.25, “listener options”](/momentum/3/3-reference/ecelerity-conf#table-listener-options) and for a separate list of cluster options see [Table 7.1, “ Cluster options ”](/momentum/3/3-reference/cluster-config-management#table-cluster-options). Options are sorted alphabetically by name. If an option functions as a scope, this is indicated by `(scope)`. The `Type` column indicates the MTA type of a given option. Options of type `na` do not directly apply to either a sending or receiving MTA. If an option has a default value, it is shown in the `Default` column followed by a version number, if there has been a change to the default value. If the value of an option cannot be changed at runtime, the `Default` column will show `(non-dynamic)`. The `Version` column shows when an option was introduced. Options that are not valid in version 3.0 or higher are not shown.
 
-<a name="All-options-table"></a> 
+<a name="All-options-table"></a>
 
 
 | Option/Description | Type | Default | Version | Scopes |
 | --- | --- | --- | --- | --- |
+| [prefer_ipv6_mxs](/momentum/3/3-reference/3-reference-conf-prefer_ipv6mxs) - Prefer IPv6 addresses over IPv4 addresses at a given MX priority level | sending | false | 3.6.12 | global |
```

Comment:
Same thing here with the alphabetic order, can you move this down so it fits with the rest?